community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/TransactionCounters.java

```diff
@@ -19,72 +19,33 @@
  */
 package org.neo4j.kernel.impl.transaction;
 
-import java.util.concurrent.atomic.AtomicLong;
-
-public class TransactionCounters implements TransactionMonitor
+public interface TransactionCounters
```

Comment:
Yes I think we do need it. Indeed having one interface for reading counters allows to us not to leak the `TransactionMonitor` methods where they are not supposed to be used.